src/Gui/Stories/Collapsible.story.luau

```diff
@@ -0,0 +1,87 @@
+--!nonstrict
+local Hoarcekat = script:FindFirstAncestor("Hoarcekat")
+local React = require(Hoarcekat.Packages.React)
+local ReactRoblox = require(Hoarcekat.Packages.ReactRoblox)
+
+local Assets = require(Hoarcekat.Plugin.Assets)
+
+local e = React.createElement
+local Collapsible = require(Hoarcekat.Plugin.Gui.Components.Collapsible)
+local IconListItem = require(Hoarcekat.Plugin.Gui.Components.IconListItem)
```

Comment:
As an addendum to my previous comment on requires--keep all requires that aren't packages together.